PhysicsTools/NanoAOD/python/triggerObjects_cff.py

```diff
@@ -218,19 +216,49 @@
   selections = selections2016
 )
 
-from PhysicsTools.PatUtils.L1ECALPrefiringWeightProducer_cff import prefiringweight
-run2_HLTconditions_2016.toModify(prefiringweight, DataEra = cms.string("2016BtoH"))
+from PhysicsTools.PatUtils.L1PrefiringWeightProducer_cff import prefiringweight
+#Next line will be updated once we get UL2016 maps
+(run2_muon_2016 & tracker_apv_vfp30_2016).toModify( prefiringweight, DataEraECAL = cms.string("2016BtoH"),  DataEraMuon = cms.string("2016preVFP"))
```

Comment:
Hello. Same comment as in the master PR:
I updated the prefiring maps for UL16 in https://github.com/cms-data/PhysicsTools-PatUtils/pull/3
Can you therefore add the following changes to pick the updated maps? 
`DataEraECAL = "UL2016preVFP"`
and on the next line:
`DataEraECAL = "UL2016postVFP"`
Thanks !